src/inference/fixpoint.rs

```diff
@@ -119,12 +130,12 @@ where
     }
 
     let (covered, size) = invariant_cover(m, conf, &fo, &proof);
-    let trivial = count_trivial(conf, &fo, &proof);
+    // let trivial = count_trivial(conf, &fo, &proof);
 
     println!("Fixpoint size = {}", proof.len());
     println!("Fixpoint runtime = {:.2}s", total_time);
     println!("Covers {covered} / {size} of handwritten invariant.");
-    println!("{trivial} out of {} lemmas are trivial.", proof.len());
+    // println!("{trivial} out of {} lemmas are trivial.", proof.len());
```

Comment:
If this is useful can you leave it in as a `log::debug`? Does it make sense to delete this line and `count_trivial`?

Reply:
This and `count_trivial` are now deleted.